internal/conductorr/integration/qbittorrent.go

```diff
@@ -6,7 +6,7 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
```

Comment:
deprecated on the go version used in the flake and the go lsp was getting mad that we were still using this